src/pages/Details.js

```diff
@@ -0,0 +1,180 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link, useNavigate, useParams } from 'react-router-dom';
+import { FaBed, FaBath } from 'react-icons/fa';
+import { MdMeetingRoom } from 'react-icons/md';
+import { FiArrowRightCircle } from 'react-icons/fi';
+import { getDetails, fullDetails } from '../redux/details/GetDetails';
+import '../styles/LoadingSpinner.css';
+
+const Details = () => {
+  const navigate = useNavigate();
+  const params = useParams();
+  const { id } = params;
+  const dispatch = useDispatch();
+  // const [isLoading, setIsLoading] = useState(true);
```

Comment:
- [ ] Kindly, remove this old piece of commented code in order to improve the repo's cleanliness. 👍